static/js/classroom/game.js

```
@@ -0,0 +1,566 @@
+function updateUsersList() {
+  const usersListElement = document.getElementById('users-list');
+  if (!usersListElement) return;
+  
+  usersListElement.innerHTML = '';
+  
+  // Get unique users from chair occupancy
+  const users = new Map();
+  
+  for (const chairId in chairsOccupancy) {
+    const chairData = chairsOccupancy[chairId];
+    if (chairData && chairData.userId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)